### PR TITLE
Create sample configuration files for existing applications

### PR DESCRIPTION
**Pull request by jbemenheiser**
The objective of the change is to create a configuration file format that allows users to save a local config file that contains their personal information without overwriting the default file in the repository.  This is accomplished by modifying the sample config (ex: application.sample.ini) with their personal information and saving it locally without the .sample in the file name (ex: application.ini)   
 
The following changes were made to all of the user configuration files (where application is a general replacement for proxy, deviceconfiguration, faraday_config, telemetry or aprs)
* The application.sample.ini files were created from the latest application.ini files.
* The application.ini files were removed
* The application.ini files were added to the .gitingnore list

The following quick start guides were updated with the new instructions.
* Tutorials/start/configuring-proxy.md
* Tutorials/start/configuring-faraday.md
* Tutorials/start/configuring-rf-faraday.md
 
The following readme files were updated with the new instructions.
* proxy/readme.md
* Applications/Telemetry/readme.md
* Applications/APRS/readme.md

The old proxy-template.ini file was removed
 
This pull request is a cleaned up version of #110   (It can be closed if this one is approved)

### CODE REVIEW

**Comment by jbemenheiser** [comment]
This pull request should close issues #94 and #98.

File: Applications/APRS/readme.md [hunk 1]
```diff
@@ -11,11 +11,13 @@ This application provides a socket link to APRS-IS servers and sends data from F
...
 # Configuration
-Before you can connect to the APRS-IS network you need to update your `aprs.ini` file with your callsign.
-
- 1. Navitage to the `Applications/APRS` folder
- 2. Open `aprs.ini` in a text editor and update the information requested
+Before you can connect to the APRS-IS network you need to create an `aprs.ini` file with your callsign.
+ 
+ 1. Navigate to the `Applications/APRS` folder
+ 2. Create a copy of `aprs.sample.ini` and rename the new file `aprs.ini`
+ 3. Open `aprs.ini` in a text editor and update the requested information noted  by `REPLACEME`
```
Comment (kb1lqc) [inline Applications/APRS/readme.md]: There is a double-space between "noted  by"

File: Applications/Telemetry/readme.md [hunk 2]
```diff
@@ -6,8 +6,21 @@ Basing Telemetry on a [Flask](http://flask.pocoo.org/) server allows it to provi
 This also allows us to use a standard web browser for basic debugging. We can easily see telemetry data with a simple URL query as we will show you shortly. [DB Browser for SQLite](http://sqlitebrowser.org/) is also a great tool which allows you to manipulate the database and perform simple plotting. It's simple to use the program to export a CSV for plotting in Excel as well.
 
 ## Configuration
+ 
+Before Telemetry can connect to Proxy and save data to an SQLite database, you need to create a `telemetry.ini` file with your callsign.
+ 
+ 1. Navigate to the `Applications/Telemetry` folder
+ 2. Create a copy of `telemetry.sample.ini` and rename the new file `telemetry.ini`
+ 3. Open `telemetry.ini` in a text editor and update the requested information noted  by `REPLACEME`
```
Comment (kb1lqc) [inline Applications/Telemetry/readme.md]: Another space between "noted  by"!

File: Tutorials/start/configuring-faraday.md [hunk 3]
```diff
@@ -11,22 +11,23 @@ The application files are located:
...
+First, we should ensure the Device Configuration program can communicate with Proxy running in the background. For this we need to create the `deviceconfiguration.ini` file from `deviceconfiguration.sample.ini` as we did for Proxy.  Next, edit the `[DEVICES]` section to contain the same configuration Proxy is configured with in `proxy.ini`
+ 
 `[FLASK]`: Flask server configuration values
  * `HOST`: IP Address or hostname of flask server
  * `PORT`: Network port to serve data
-
-
+ 
+ 
 `[DEVICES]`: Unit 0 Proxy configuration values section
```
Comment (kb1lqc) [inline Tutorials/start/configuring-faraday.md]: Non-blocking as this is an error from outside this PR. Would be good to remove reference to Unit 0 Proxy configuration as this is all unit configuration. Reword as you see fit if you like.

File: Tutorials/start/configuring-proxy.md [hunk 4]
```diff
@@ -4,11 +4,23 @@
...
 
 ## Configuring Proxy
+>This guide will walk you through these steps for the proxy configuration file.  The config files for other applications are modified in the same way.
+
+### Configuration Files
+Some applications require user specific information in order to operate.  Most commonly, `CALLSIGN` and `NODEID` are required so the software knows which radio to communicate with.  Each application that needs user information looks for it in a configuration file.  In general, a given application `ex: application.py` it will gather configuration information from the *.ini* file in the same folder with the same name as the application `ex: application.ini`.  Since these configuration files contain user specific information, they are listed in the .gitignore file to prevent contributors from committing their personal configs to the repository by accident.  For each application, a default file is stored in the git repository  with *.sample* added to the file name before the extension `ex: application.sample.ini`.  This file contains all of the required fields with either default information or  `REPLACEME` noting that user information is required.
```
Comment (kb1lqc) [inline Tutorials/start/configuring-proxy.md]: Double-space at beginning of sentence. Let's use single space.

File: proxy/readme.md [hunk 5]
```diff
@@ -12,6 +12,21 @@ A flask server runs in the main process which provides a RESTful interface for t
...
+Before Proxy can connect to any radio hardware, it needs to be configured using the `proxy.ini` file with your callsign and the serial port.
+ 
+ 1. Navigate to the `proxy` folder
+ 2. Create a copy of `proxy.sample.ini` and rename the new file `proxy.ini`
+ 3. Open `proxy.ini` in a text editor and update the requested information noted  by `REPLACEME`
+ 
+Configuring Proxy consists of changing three parameters for basic use:`CALLSIGN`, `NODEID` and `COM`. These will let Telemetry properly query Proxy to obtain the correct telemetry data.  
+ 
+* `[UNIT0]` Telemetry application section
+ * `CALLSIGN` Quantity of Faraday radios connected to computer
```
Comment (kb1lqc) [inline proxy/readme.md]: Callsign description is incorrect, in-fact, all of these descriptions for callsign, nodeid, com are off by one.

**Review by kb1lqc — CHANGES_REQUESTED** [review]
Pull request just needs some quick fixes. Otherwise it looks great! Please fix them and I will approve and merge.

**Comment by jbemenheiser** [comment]
@kb1lqc, Have a gander at the latest commit with corrections for your comments.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -106,4 +106,16 @@ All Faraday Python code is styled using [PEP8](https://www.python.org/dev/peps/p
 Documentation is mandatory to ensure that all contributors clearly understand how to use, enhance, and fix additions to the project. Faraday uses [Sphinx](http://www.sphinx-doc.org/en/stable/index.html) to provide dynamic documentation and is hosted on [Faraday's Read The Doc's Page](http://faraday-software.readthedocs.io/en/latest/).
 
 * [Sphinx Documentation](http://www.sphinx-doc.org/en/stable/contents.html)
-* [Examples Of Good Python Docstrings](http://www.sphinx-doc.org/en/stable/ext/example_google.html)
\ No newline at end of file
+* [Examples Of Good Python Docstrings](http://www.sphinx-doc.org/en/stable/ext/example_google.html)
+
+##Configuration files
+
+For any application that requires user specific configuration, add a sample configuration file with `.sample` added to the file name before the `.ini`.  Use proxy as an example where `proxy.py` uses configuration from `proxy.ini`, but `proxy.sample.ini` is stored in git and `proxy.ini` is in .gitignore.  The end user needs to create their own local copy of the configuration file that contains their information by editing the file and renaming it without the `.sample`.
```
Comment (kb1lqc) [inline CONTRIBUTING.md]: Just realized there are double-spaces for each sentence here too. Let's remove them to keep consistency.

**Review by kb1lqc — CHANGES_REQUESTED** [review]
One more comment. Looking good after that.

**Review by kb1lqc — APPROVED** [review]
Looks great! Let's do this.

**Pull request merged**